### PR TITLE
[K8s hosting] Refresh and address older issues

### PR DESCRIPTION
**Pull request by hhunter-ms**
## Description

Address items in linked issues
Refresh docs

## Issue reference

PR will close: 
#1168 
#1072

### CODE REVIEW

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md [hunk 1]
```diff
@@ -32,20 +41,29 @@ Deploying and running a Dapr enabled application into your Kubernetes cluster is
...
 
 The Dapr control-plane and sidecar images come from the [daprio Docker Hub](https://hub.docker.com/u/daprio) container registry, which is a public registry.
 
-For information about pulling your application images from a private registry, reference the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). If you are using Azure Container Registry with Azure Kubernetes Service, reference the [AKS documentation](https://docs.microsoft.com/azure/aks/cluster-container-registry-integration).
+For information about:
+- Pulling your application images from a private registry, reference the [official Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). 
+- Using Azure Container Registry with Azure Kubernetes Service, reference the [AKS documentation](https://docs.microsoft.com/azure/aks/cluster-container-registry-integration).
 
-## Quickstart
+## Stopping the cluster in development
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md]: You can remove this, since this addressed with the Dapr Injector watchdog that can be enabled on a local development cluster. I

Comment (msfussell) [reply daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md]: I meant you can remove this whole paragraph you added, not just the heading

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md [hunk 2]
```diff
@@ -32,20 +41,29 @@ Deploying and running a Dapr enabled application into your Kubernetes cluster is
...
 
-You can see some examples [here](https://github.com/dapr/quickstarts/tree/master/tutorials/hello-kubernetes) in the Kubernetes getting started quickstart.
+In some Kubernetes clustering environments, the cluster can be stopped and then restarted to save on costs. For example, the app pods start before the Dapr sidecar injector pod runs, so no sidecars are injected to the apps. Since Kubernetes doesn't enable you to define dependencies, an operator performs a Kubernetes app rollout command on the apps to restart the pods after the sidecar injector pod is running.
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md]:
```suggestion
```

File: daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-aks.md [hunk 3]
```diff
@@ -4,53 +4,53 @@ title: "Setup an Azure Kubernetes Service (AKS) cluster"
 linkTitle: "Azure Kubernetes Service (AKS)"
 weight: 2000
 description: >
-  How to setup Dapr on an Azure Kubernetes Cluster.
+  How to setup Dapr on an Azure Kubernetes Cluster
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-aks.md]: This description is not related to the content of the page. The content of the page is pure AKS setup.

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md [hunk 4]
```diff
@@ -8,111 +8,111 @@ aliases:
...
+{{% /alert %}}
 
-### Create cluster
+{{< tabs "Dapr CLI" "Helm" >}}
+ <!-- Dapr CLI -->
+{{% codetab %}}
+## Install with Dapr CLI
 
-You can install Dapr on any Kubernetes cluster. Here are some helpful links:
+You can install Dapr to a Kubernetes cluster using the [Dapr CLI]({{< ref install-dapr-cli.md >}}).
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md]:
```suggestion
You can install Dapr on a Kubernetes cluster using the [Dapr CLI]({{< ref install-dapr-cli.md >}}).
```

**Review by msfussell — APPROVED** [review]
LGTM
